### ISSUE TITLE
Excellent work! Have some questions to inquire about the GRPO

### ISSUE THREAD

**Issue by tingxueronghua** [meituan-longcat/LongCat-Video#18]
Thanks for your excellent work! I am also trying to implement flow-group-like algorithms, but I found it is not that stable under different hyper-parameters. I feel really attracted when I see your Group size is only 4, which means the algorithm is much better than the original one. I have some questions to inquire about and hope to learn from your work.

1. In this [issue](https://github.com/meituan-longcat/LongCat-Video/issues/3), you have said that "As an initial empirical fix, we found that simply scaling the policy loss by a factor of 100 enabled successful training.". I want to inquire whether the final algorithm improved a lot than the empirical version. (Because I am not sure how to implement your final version.)

2. I also noticed that you said "Our initial application of GRPO to our pre-trained model for image training proceeded smoothly.". Want to inquire whether you have applied the improvement mentioned in your paper. Also want to inquire about the group size for image training... Because I found the original flow-grpo / flow-grpo-fast seems to be really hard to increase the rewards, and I do not have much idea which hyper-parameters are essential.

**Comment by tingxueronghua** [comment]
Many thanks!
